src/frontend/src/features/ui/components/transient-tooltip/_index.scss

```
@@ -0,0 +1,24 @@
+.transient-tooltip__wrapper {
+    display: inline-flex;
+}
+
+// Overrides on top of Cunningham's `.c__tooltip`: switch to fixed positioning
+// so we can place it at viewport coordinates from JS, mark non-interactive,
+// and centre horizontally on the trigger's mid-axis.
+.transient-tooltip {
+    --offset: 8px;
+    position: fixed;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)